server/server.js

```diff
@@ -1,83 +1,91 @@
 const express = require('express');
 const cors = require('cors');
 require('dotenv').config();
+//add path 
+const path = require('path');
 const db = require('./db/db-connection.js');
+const affirmationApi = require('./routes/affirmationapi.js');
+
 
 const app = express();
+//after express but before port - connecting the directory name which is found in client, so go back once from here in server
+//up to the project folder, then go to client, then go to build 
+const REACT_BUILD_DIR = path.join(__dirname, '..', 'client', 'build');
+app.use(express.static(REACT_BUILD_DIR));
 
-const PORT = 8080;
+//calling either the process.env.PORT or 8080 if that's not available
+const PORT = process.env.PORT || 8080;
 app.use(cors());
 app.use(express.json());
 
+//needed to install node-fetch@2 in order to do fetch on affirmationsapi.js
+app.use('/random', affirmationApi);
 // creates an endpoint for the route /api
 app.get('/', (req, res) => {
-  res.json({ message: 'Hello from My template ExpressJS' });
+  //res.json({ message: 'Hello from My template ExpressJS' }); Instead of the response being a json it's a 
+  res.sendFile(path.join(REACT_BUILD_DIR, 'index.html'));
 });
 
+//routes
+const random =require( "./routes/affirmationapi.js");
+app.use("/random", random)
+
 // create the get request
-app.get('/api/students', cors(), async (req, res) => {
-  // const STUDENTS = [
-
-  //     { id: 1, firstName: 'Lisa', lastName: 'Lee' },
-  //     { id: 2, firstName: 'Eileen', lastName: 'Long' },
-  //     { id: 3, firstName: 'Fariba', lastName: 'Dadko' },
-  //     { id: 4, firstName: 'Cristina', lastName: 'Rodriguez' },
-  //     { id: 5, firstName: 'Andrea', lastName: 'Trejo' },
-  // ];
-  // res.json(STUDENTS);
+app.get('/table', cors(), async (req, res) => {
+  
   try {
-    const { rows: students } = await db.query('SELECT * FROM students');
-    res.send(students);
+    const { rows: affirmations } = await db.query('SELECT * FROM affirmations');
+    res.send(affirmations);
   } catch (e) {
     return res.status(400).json({ e });
   }
 });
 
 // create the POST request
-app.post('/api/students', cors(), async (req, res) => {
-  const newUser = {
-    firstname: req.body.firstname,
-    lastname: req.body.lastname,
-  };
-  console.log([newUser.firstname, newUser.lastname]);
-  const result = await db.query(
-    'INSERT INTO students(firstname, lastname) VALUES($1, $2) RETURNING *',
-    [newUser.firstname, newUser.lastname],
-  );
-  console.log(result.rows[0]);
-  res.json(result.rows[0]);
-});
+// app.post('/api/students', cors(), async (req, res) => {
```

Comment:
It's fine to leave this in for now, if you're going to use it as a reference for how you might add to your affirmations table